webrtc-stats.html

```diff
@@ -1577,6 +1579,33 @@ <h2>
                   the configuration is considered power efficient.
                 </p>
               </dd>
+              <dt>
+                <dfn>totalAssemblyTime</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  Only [= map/exist =]s for video. The sum of the time, in seconds, each video frame takes
+                  from the time the first RTP packet is received (reception timestamp) and to the time
+                  the last RTP packet of a frame is received.
```

Comment:
Only applicable to frames which consist of multiple packets?

Reply:
copied text from below after moving around which makes it a bit better